doc/source/whatsnew/v2.1.0.rst

```diff
@@ -97,8 +97,10 @@ Other enhancements
 - Let :meth:`DataFrame.to_feather` accept a non-default :class:`Index` and non-string column names (:issue:`51787`)
 - Performance improvement in :func:`read_csv` (:issue:`52632`) with ``engine="c"``
 - :meth:`Categorical.from_codes` has gotten a ``validate`` parameter (:issue:`50975`)
+- Adding ``engine_kwargs`` parameter to :meth:`DataFrame.to_excel` (:issue:`53220`)
 - Performance improvement in :func:`concat` with homogeneous ``np.float64`` or ``np.float32`` dtypes (:issue:`52685`)
 - Performance improvement in :meth:`DataFrame.filter` when ``items`` is given (:issue:`52941`)
+-
```

Comment:
```suggestion
```

Reply:
Should be fixed with most recent push.

Reply:
@mroeschke Adding the blank line back. Looks like it was needed, unless I am misunderstanding something

![image](https://github.com/pandas-dev/pandas/assets/45905457/8dc5b126-7b40-482b-918a-3b8855f6166a)

Reply:
@mroeschke request was to remove the line `- `, not the blank line. We mostly try to avoid unnecessary lines in the diff. However, most sections end with `- `; I'm okay with this if @mroeschke is.

Reply:
Sure we can keep this around for now. Not a big deal